### PR TITLE
`Multi::zipFilled()`, `Stream::zipFilledWith()` methods added

### PR DESCRIPTION
**Pull request by Smoren**
Hi @markrogoyski,

New methods:
* `Multi::zipFilled()`;
* `Stream::zipFilledWith()`.

### CODE REVIEW

File: src/Multi.php [hunk 1]
```diff
@@ -58,6 +59,30 @@ public static function zipLongest(iterable ...$iterables): \Generator
...
+     *
+     * @param mixed $filler
+     * @param iterable<mixed> ...$iterables
+     *
+     * @return \Generator<array<mixed>>
+     */
+    public static function zipFilled($filler, iterable ...$iterables): \Generator
+    {
+        $iterator = new JustifyMultipleIterator(...$iterables);
+        $iterator->setFiller($filler);
```
Comment (markrogoyski) [inline src/Multi.php]: Suggestion: Perhaps consider a new "Filled" iterator or something that would take the filler as a constructor parameter along with the iterables. Using a setter mutates the iterator. It is generally a good idea to consider objects as immutable whenever possible. While an iterator technically does have mutable state with its internal pointer, we can avoid additional unnecessary mutations by requiring all parameters at the time of construction.

**Comment by markrogoyski** [comment]
Hi @Smoren,

Thank you for the PR implementing zipFilled. Just one small suggestion about how to construct the iterator.

Thanks,
Mark

File: src/Multi.php (hunk 1 above)
Comment (Smoren) [reply src/Multi.php]: Done.

**Comment by Smoren** [comment]
Thank you @markrogoyski,

I've added `$filler` param to the constructor interface of `JustifyMultipleIterator` class and removed `setFiller()` method.

**Pull request merged**